### DISCUSSION TITLE
In which intervals does the background job for Facerecognition run? How to adjust?

### DISCUSSION THREAD

**Discussion by JeCheeseSmith** [nextcloud/all-in-one#4759] [Questions] [answered]
Hi!

I'm trying to get the facerecognition app running in a smoother way.  I run my instance on a VPS with low resources (4GB Ram, 2 Cores)

I noticed the process doesn't go so fast, which is normal considering the hardware. However, I noticed if I let the background job run mannualy for a while it recognises far more faces. (Same settings)

Thats why I'm wondering in  which time intervals the background job gets executed or restarted? (E.g. what is the timeout?)

Morely, how could I adjust the timeout for the backgroundjob or adjust the cronjob (If desirable?) ?

**Comment by szaimen** [comment]
> In which intervals does the background job for Facerecognition run?

Hi @matiasdelellis do you know the answer?

**Comment by matiasdelellis** [comment, marked as answer]
Hi both,

Taking into account the philosophy of Nextcloud AIO, at installation time it starts without any time window. This has no consideration of resources or time.. Unfortunately it can take a while. 😞 

https://github.com/nextcloud/all-in-one/blob/main/community-containers/facerecognition/facerecognition.json#L31C1-L31C81

Use 'face:progress' to estimate that time..

After that, you must schedule the task according to your needs.
* https://github.com/matiasdelellis/facerecognition/wiki/Schedule-Background-Task
